services/webhook/discord.go

```diff
@@ -101,6 +101,13 @@ var (
 	redColor         = color("ff3232")
 )
 
+// https://discord.com/developers/docs/resources/message#embed-object-embed-limits
+// Discord has some limits in place for the embeds.
+// According to some tests, there is no consistent limit for different characet sets.
```

Comment:
Sorry I made a typo, will fix

```suggestion
// According to some tests, there is no consistent limit for different character sets.
```